docs/how to/use-transferbench.rst

```diff
@@ -1,106 +1,146 @@
 .. meta::
-  :description: TransferBench documentation 
-  :keywords: TransferBench, API, ROCm, documentation, HIP
+  :description: TransferBench is a utility to benchmark simultaneous transfers between user-specified devices (CPUs or GPUs)
+  :keywords: Using TransferBench, TransferBench Usage, TransferBench How To, API, ROCm, documentation, HIP
 
+.. _using-transferbench:
 
+---------------------
 Using TransferBench
 ---------------------
-  
-Users have control over the SRC and DST memory locations by indicating memory type followed by the device index. TransferBench supports the following:
 
-* coarse-grained pinned host memory
-* unpinned host memory
-* fine-grained host memory
-* coarse-grained global device memory
-* fine-grained global device memory
-* null memory (for an empty transfer).
+You can control the SRC and DST memory locations by indicating the memory type followed by the device index. TransferBench supports the following memories:
+
+* Coarse-grained pinned host memory
```

Comment:
you can remove the term "memory" from this bulleted list. Redundant with the intro in play

Reply:
Done